lib/xlint.rb

```diff
@@ -101,7 +111,7 @@ def check_deployment_target(change)
           path: change[:file],
           position: change[:line_number],
           message: 'Deployment target changes should be approved by the team lead.',
-          severity: 'error'
+          severity: 'warn'
```

Comment:
Gergich will use the lowest value in the draft as the score it posts. On the Jenkins job I'm testing with, having this set to error prevented Gergich from posting to the Lint-Review column.